### DISCUSSION TITLE
How well does TiddlyWiki server scale?

### DISCUSSION THREAD

**Discussion by Ca1vino** [TiddlyWiki/TiddlyWiki5#5608] [Q&A]
Thanks for the answers I've received so far...

I used and modified TW classic extensively, and only recently started migrating and redoing customizations within TW5. 

Now I'm evaluating certain project ideas, and one of them would potentially involve Node.js wikis with large numbers of tiddlers. A question is, how well does it scale?  

With consumer hardware (laptops, desktop systems) would it support 25,000 tiddlers?  How about 100,000? If this is difficult, are there any configuration strategies or simple modifications that would make it possible to serve out of a large set of infrequently accessed tiddlers?

**Comment by andjar** [comment]
Some information can be found in the [documentation](https://tiddlywiki.com/#Performance) and [this reddit thread](https://www.reddit.com/r/TiddlyWiki5/comments/itpqrc/how_much_tiddlers_tiddlywiki_can_handle/) is fairly recent, linking to a tiddlywiki with [63 000 tiddlers](https://marxsal.github.io/EODICT/EO-DICT.html). However, they seem to focus on client rather than server
